app/src/main/java/org/p2p/wallet/common/ResourcesProvider.kt

```diff
@@ -0,0 +1,17 @@
+package org.p2p.wallet.common
+
+import android.content.Context
+import android.content.res.Resources
+import androidx.annotation.ColorRes
+import androidx.annotation.StringRes
+
+class ResourcesProvider(private val context: Context) {
```

Comment:
I think this class is redundant, cause we can have just extensions, i guess we already them. So which problems resolve this class ?

Reply:
mocks / stubs in unit tests?

Reply:
In some cases we need resources in presenters, but our extensions are expected to be used for Views. 
Thus, there is no other way that including context/resources/resources provider in presenter. 

The only thing I was confused is injecting the theme because getting colors via resources requires Theme right now. 
We decided to find some way, where we don't think about theme injection.